_pages/do-i-need-clearance/form-updates-and-conversions.md

```diff
@@ -1,3 +1,18 @@
 ---
-title: Form updates and conversions
+title: Converting forms from paper to digital
+permalink: /form-updates-and-conversions.md/
```

Comment:
@amandacostello I figured it out! Adding this `permalink` I believe is causing the page to not show in the side nav. Can you remove and retest?